llvm/include/llvm/BinaryFormat/Minidump.h

```diff
@@ -32,6 +32,10 @@ LLVM_ENABLE_BITMASK_ENUMS_IN_NAMESPACE();
 struct Header {
   static constexpr uint32_t MagicSignature = 0x504d444d; // PMDM
   static constexpr uint16_t MagicVersion = 0xa793;
+  // We set all the high bits flag to indicate this is from LLDB.
+  // We don't want to conflict with anything Microsoft is using the flags for
+  // and the highest bits are not currently being used.
+  static const uint32_t LLDB_HEADER_FLAG = 0xF0000000;
```

Comment:
uint64_t to match the width of the field, and change the name to fit the convention of the constants above.